packages/daemon/src/web-page.js

```diff
@@ -65,6 +65,8 @@ const hardenedEndowments = harden({
   TextEncoder,
   TextDecoder,
   URL,
+  // This one is important for exo to work.
```

Comment:
@mhofman or @michaelfig can I ask you to help @kumavis expand this comment to explain why and where to look for more? I think this is the work of the handled promise shim, but I did not know it replaced `globalThis.Promise`.

Reply:
`@endo/eventual-send` does not mutate `Promise` in any way, so I'm confused as to why this line is necessary.

Reply:
I really don't understand why this is necessary. Is there anything in your environment that might be replacing the global `Promise`? Maybe some telemetry library?